extensions/wikia/AdEngine/js/video/player/playerTracker.js

```diff
@@ -59,7 +59,8 @@ define('ext.wikia.adEngine.video.player.playerTracker', [
 				'browser': [ browserDetect.getOS(), browserDetect.getBrowser() ].join(' '),
 				'additional_1': canFloat,
 				'additional_2': floatingState,
-				'vast_id': params.vastId || emptyValue.string
+				'vast_id': params.vastId || emptyValue.string,
+				'video_id': (adContext.get('targeting.featuredVideo') || {}).mediaId
```

Comment:
It won't work for consecutive videos. We should take current mediaId from player